_opensearch/snapshots/snapshot-management.md

```diff
@@ -0,0 +1,100 @@
+
+The cluster’s global metadata must be readable. To include an index in a snapshot, the index and its metadata must also be readable.
+{: .note}
+
+An SM schedule is a custom [cron]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron) expression that specifies the frequency of taking snapshots, and the timing of creating and deleting snapshots. 
+
+An SM configuration includes the indices and repository for these snapshots, and has all parameters you can define when [creating a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshot-restore#take-snapshots) manually. 
+
+An SM configuration can have more than one index.
+{: .note}
+
+## Performance 
+
+One snapshot can contain as many indices as there are in the cluster. We expect at most dozens of SM policies in one cluster, but a snapshot repository can safely scale to thousands of snapshots. However, to manage its metadata, a large repository requires more memory on the cluster manager node. 
+
+Snapshot management depends on the job scheduler plugin to schedule a job that is run periodically. Each SM policy corresponds to one SM scheduled job. The scheduled job is lightweight, so the burden of SM depends on the snapshot creation frequency and the burden of running snapshot operation itself. However, too many pending SM tasks can block other high-priority tasks in the cluster.
+
+## Concurrency 
+
+- Taking a snapshot doesn’t block indexing or other requests. However, the snapshot won’t include changes made after the snapshot process starts. 
+
+- Snapshot deletion doesn't block snapshot creation.
+
+- Taking a snapshot can temporarily pause shard allocations. 
+
+We don't recommend creating several SM policies with the same schedule and overlapping indices in one cluster, since it hinders performance. 
+{: .warning }
+
+We don't recommend setting up the same repository for multiple SM policies in different clusters, since it hinders performance. 
+{: .warning }
+
+## Failure management
+
+If snapshot creation fails, it is retried for a maximum of three times. The failure message is saved into metadata and is removed if a subsequent snapshot creation attempt succeeds. Possible reasons for failure include red indices, and shard reallocation.
+
+## Security
+
+SM follows the OpenSearch plugin security model:
+
+- The system index is only accessible to super admin (who uses the cluster SSL credential directly). Even the admin user cannot access the system index.
+    
+- The security plugin checks each added API plugin at transport layer. The security plugin checks the security roles of the calling user if the roles contain the permission associated with this API.
```

Comment:
The security plugin checks each API call at transport layer.

Reply:
Removed